gnome/gnome-radio/Portfile

```diff
@@ -3,21 +3,21 @@
 PortSystem          1.0
 
 name                gnome-radio
-version             0.4.0
+version             16.0.42
 revision            0
 set branch          [join [lrange [split $version .] 0 1] .]
 
 categories          gnome
 platforms           darwin
 license             GPL-3+
 maintainers         {gnome.org:ole @oleaamot} openmaintainer
-description         GNOME Radio
-long_description    GNOME Radio is the Public Network Radio Software for Accessing Free Audio Broadcasts from the Internet on GNOME.
+description         Radio
```

Comment:
The single word "Radio" seems to be a poor description.

Reply:
Fixed description in the latest pull request against HEAD.